crates/environ/src/module.rs

```diff
@@ -1144,3 +1170,19 @@ pub struct InstanceSignature {
     /// The name of what's being exported as well as its type signature.
     pub exports: IndexMap<String, EntityType>,
 }
+
+/// Type information about functions in a wasm module.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub struct FunctionType {
+    /// The type of this function, indexed into the global type tables for
+    /// a module compilation.
```

Comment:
"Global" in the comment here is a little ambiguous; would it be correct to say "module-wide" instead?